samples/simple_private_memory/simple_private_memory.cpp

```diff
@@ -60,33 +58,36 @@ int main() {
     buffer<int, 1> buf(data, range<1>(nItems));
 
     /* This command group enqueues a kernel on myQueue
-     * that adds the thread id to each element of the
-     * data array. */
+     * that adds the work-item id to each element of the
+     * data array. Effectively, it creates an array of
+     * consecutive integers. */
     myQueue.submit([&](handler& cgh) {
       auto ptr = buf.get_access<access::mode::read_write>(cgh);
       /* We create a linear (one dimensional) group range, which
-       * creates a thread per element of the vector. */
+       * creates a work-item per element of the vector. */
       auto groupRange = range<1>(nItems / nLocals);
```

Comment:
You can even replace all the `()` by `{}` in the constructors to be nicer and more modern.